src/pages/Account/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import {useState} from "react";
+import {Tabs, Tab, Box} from "@mui/material";
+import RawDataTab from "./RawDataTab/Index";
+import TransactionTab from "./TransactionsTab/Index";
+import {assertNever} from "../../utils";
+
+const TabComponents = Object.freeze({
+  transactions: TransactionTab,
+  rawData: RawDataTab,
+});
+
+type TabValue = keyof typeof TabComponents;
+
+const TAB_VALUES: TabValue[] = ["transactions", "rawData"];
+
+type TabPanelProps = {
+  value: TabValue;
+  address: string;
+};
+
+function TabPanel({value, address}: TabPanelProps): JSX.Element {
+  switch (value) {
```

Comment:
This could be simplified to:

```jsx
function TabPanel({value, address}: TabPanelProps): JSX.Element {
  const TabComponent = TabComponents[value];
  return <TabComponent address={address} />;
}
```